### ISSUE TITLE
int[] syntax sugar for array(int) constructor

### ISSUE THREAD

**Issue by mhagmajer** [xFAANG/askql#51]

**Comment by Tuhaj** [comment]
We need to add here more description, it is hard to find out the scope of the task and not clear where to begin @mhagmajer

**Comment by czerwinskilukasz1** [comment]
The scope of this task is to adapt the pegjs parser to accept the new 'int[]' syntax in addition to the current one. 
As for the details of the syntax change proposal, we need to ask for Marcin's explanation and examples.

**Comment by Fifciu** [comment]
I could take care of that. I understand it is Peg.js related - I browsed code. However, I need some tips and methods to test it well.

**Comment by mhagmajer** [comment]
@czerwinskilukasz1 can you help here?

**Comment by Fifciu** [comment]
I am taking it

**Comment by czerwinskilukasz1** [comment]
Great, congrats, Filip :) 
Feel free to ask questions here or contact with us on our [Discord channel](https://discord.gg/pYdzypH).

**Comment by czerwinskilukasz1** [comment]
Filip, by the way, didn't you want to work on https://github.com/xFAANG/askql/issues/52 instead? ;) @Fifciu

**Comment by Fifciu** [comment]
Nope. However, as both are about synthetic sugar - maybe it will be my next step :)

![image](https://user-images.githubusercontent.com/30155292/85407503-94240600-b563-11ea-8f85-d66a01646132.png)

**Comment by czerwinskilukasz1** [comment]
Cool :)

**Comment by Tuhaj** [comment]
Taking over!

**Comment by czerwinskilukasz1** [comment]
Cool, @Tuhaj

**Comment by czerwinskilukasz1** [comment]
Hi @Fifciu , would you like to take a look at this issue? :)

**Comment by czerwinskilukasz1** [comment]
@Tuhaj , it's been a couple of weeks since you were assigned.  Any thoughts or progress on this one?
